content/docs/command-reference/exp/remove.md

```diff
@@ -55,4 +59,10 @@ $ dvc exp remove exp-1dad0 exp-1df77
 $ dvc exp list
 master:
         exp-e6c97
+        exp-23d5a
+$ dvc exp remove -A
+
+$ dvc exp list
+
+$
 ```
```

Comment:
This needs explaining and removing some of the last lines please @karajan1001

Reply:
How to fix this?

Reply:
````suggestion
$ dvc exp list
master:
        exp-e6c97
        exp-23d5a

$ dvc exp remove -A

$ dvc exp list
```

Nothing is listed after the last `dvc exp list` because they're all gone.
````